scribbler/tests/views.py

```diff
@@ -0,0 +1,22 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth import login
+from django.contrib.auth.forms import AuthenticationForm
+
+
+def testpage(request):
```

Comment:
Could this just use the built-in `login` view from contrib auth? https://docs.djangoproject.com/en/1.8/topics/auth/default/#django.contrib.auth.views.login